docs/methods.md

```diff
@@ -1,20 +1,20 @@
 # Overview
 
-Each publiq API supports one or more authentication methods, depending on the security that it requires.
+Each publiq API supports one of three authentication methods, depending on the security that it requires.
 
 ## When to use which method
 
 Which authentication method you need to use will in the first place be **determined by which API endpoint(s) you want to access**.
 
-There are 5 possible scenarios for an endpoint:
+There are 3 possible scenarios for an endpoint:
 
 1.  The endpoint requires **no authentication** at all. For example fetching the JSON representation of a specific event from UiTdatabank.
 2.  The endpoint requires **[client identification](#client-identification)**. You will only need to include your client id in the request and you don't need a token. For example Search API.
-3.  The endpoint accepts **both [user access tokens](#user-access-tokens) or [client access tokens](#client-access-tokens)**. You can pick whatever token type is best suited to your situation. See their respective documentation for more info.
-4.  The endpoint requires [**client** access tokens](#client-access-tokens) *specifically*. You will need to fetch a token using your client id and secret from a backend.
-5.  The endpoint requires [**user** access tokens](#user-access-tokens) *specifically*. You will need to let your end user log in through UiTID and use the resulting token.
+3.  The endpoint requires a **[token](#tokens)**. You can pick whatever token type is best suited to your situation (almost every endpoint will support both types). See their respective documentation for more info.
```

Comment:
we zeggen 'both types' maar de details dat er 2 zijn staan pas verderop. Ofwel zou ik dat weglaten, ofwel zeggen we iets als "You can pick a user or client token depending on your situation ..."